src/gui/updater/ocupdater.cpp

```diff
@@ -104,15 +104,23 @@ bool OCUpdater::performUpdate()
     QString updateFile = settings.value(updateAvailableC).toString();
     if (!updateFile.isEmpty() && QFile(updateFile).exists()
         && !updateSucceeded() /* Someone might have run the updater manually between restarts */) {
-        const QString name = Theme::instance()->appNameGUI();
-        if (QMessageBox::information(nullptr, tr("New %1 Update Ready").arg(name),
-                tr("A new update for %1 is about to be installed. The updater may ask\n"
-                   "for additional privileges during the process.")
-                    .arg(name),
-                QMessageBox::Ok)) {
+        const auto messageBoxStartInstaller = new QMessageBox(QMessageBox::Information,
+            tr("New %1 Update Ready").arg(Theme::instance()->appNameGUI()),
```

Comment:
@jancborchardt told me once we never use capitalization in titles. So should be "New %1 update ready"

Reply:
@FlexW I don't want to change this now. This will require translation. So, let's use the old text for now. Or, should we ask Jan's opinion? @jancborchardt Do you think I should change existing strings?

Reply:
I get your point, but why not change it now. At some point, we need to change it. If we do not do it now, it may never happen.

Reply:
Yeah, agree with @FlexW here – better to change things when we find them. :) (Also cause there’s probably not so many wrong anyway.)

Reply:
@FlexW fixed